### PR TITLE
Introduce package structure

### PR DESCRIPTION
**Pull request by ckreibich**
It's way easier to unit-test the client in package structure, so I'm adding that here. This only introduces the modules — it does not provide setup.py etc for a PyPI-compatible package. There's currently no way to install the Broker dependency as a package, so there's not much point. CMakeLists.txt simply deploys the files along the other Zeek-based Python modules.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -2,25 +2,38 @@ cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
...
+    ${CMAKE_CURRENT_BINARY_DIR}/zeek-client @ONLY)
 install(DIRECTORY DESTINATION bin)
-install(PROGRAMS ${configed_file} DESTINATION bin)
+install(PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/zeek-client DESTINATION bin)
+
+# Install the Python module tree
+install(DIRECTORY DESTINATION ${PY_MOD_INSTALL_DIR})
+install(DIRECTORY zeekclient DESTINATION ${PY_MOD_INSTALL_DIR})
+
+# Install the constants file with upated content:
```
Comment (timwoj) [inline CMakeLists.txt:25]:
```suggestion
# Install the constants file with updated content:
```

File: zeekclient/__init__.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import logging
```
Comment (timwoj) [inline zeekclient/__init__.py:1]: I don't see `logging` getting used in this file, so this import might not be necessary.

**Review by timwoj — APPROVED** [review]
Just some minor questions.

File: zeekclient/__init__.py (hunk 2 above)
Comment (ckreibich) [reply zeekclient/__init__.py:1]: You're faster than my upcoming pylinting. :-) Thanks!

File: zeek-client [hunk 3]
```diff
@@ -68,1429 +41,44 @@ ZEEK_PYTHON_DIR = '@PY_MOD_INSTALL_DIR@'
...
 
-    parser = create_parser()
     args = parser.parse_args()
 
-    # After we've parsed the command line, finalize config settings in the
-    # expected hierarchical order:
-    ZC_CONFIG.update_from_file(args.configfile)
-    ZC_CONFIG.update_from_env()
-    ZC_CONFIG.update_from_args(args)
+    # Finalize config settings in expected hierarchical order:
```
[the remark this answers is not in the document]
Comment (ckreibich) [reply zeek-client:60]: Yeah was wondering too — there's currently an additional `update_from_env()` a little earlier in the file, so I figured it's just easiest to leave them individually visible for now.

File: CMakeLists.txt (hunk 1 above)
Comment (ckreibich) [reply CMakeLists.txt:25]: Thanks!

**Pull request merged**